### DISCUSSION TITLE
James Tenney's A Rose Is A Rose Is A Round neoscore Formalization

### DISCUSSION THREAD

**Discussion by jgarte** [DigiScore/neoscore#46] [Ideas]
![2022-06-22-190245_803x558_scrot](https://user-images.githubusercontent.com/47760695/175180707-454ac37b-1ca0-4eab-ba58-f63eb174693b.png)

This one might be a good example for using the [western notation primitives](https://neoscore.org/western/western.html).

**Comment by ajyoon** [comment]
Path and text deformations is actually one of the big things Neoscore doesn't have good support for right now, so this would be pretty tricky to implement. We'd have to implement our own path tesselation+deformation system for it, which I believe is pretty involved. I agree though this would be a very powerful capability if anyone had the time to invest into it.
